### PR TITLE
src: make callback option in modal

### PR DESCRIPTION
**Pull request by jazzdan**
Screenshot from 2020-08-06 15-15-18

Still needs a bit of styling love, paging @hyu :) but it works!

### CODE REVIEW

File: src/_includes/index_cta.html [hunk 1]
```diff
@@ -1,18 +1,88 @@
 <section class="modal" data-modal="one">
-  <h2 class="modal__title">
-    Schedule a Demo
-  </h2>
-  <button class="modal__close">{% svg assets/svg/x.svg %}</button>
-  <div class="modal__content">
-    <div class="calendly-inline-widget" data-url="https://calendly.com/dbentley/tilt-enterprise?hide_event_type_details=1">
+  <div class="calendly">
+    <h2 class="modal-title">
```
Comment (hyu) [inline src/_includes/index_cta.html:3]: FYI FWIW: I changed all the `modal__` prefixes to `modal-` just to make it more consistent with the off-brand usage of BEM elsewhere in this project. ¯\_(ツ)_/¯

File: src/_includes/index_cta.html [hunk 2]
```diff
@@ -1,18 +1,88 @@
...
+          <label for="drip-name">Name</label>
+          <input class="modal-form-input" type="text" id="drip-name" name="fields[name]" value="" />
+        </div>
+        <div class="modal-form-row">
+          <label for="drip-email">Email</label>
+          <input class="modal-form-input" type="email" id="drip-email" name="fields[email]" value="" />
+        </div>
+        <div class="modal-form-row">
+          <label for="drip-Phone">Phone</label>
+          <input class="modal-form-input" type="tel" id="drip-Phone" name="fields[Phone]" value="" />
```
Comment (hyu) [inline src/_includes/index_cta.html:50]: FYI: Also changed this type from "text" to "tel," though I'm not sure how important that is.

File: src/_sass/home.scss [hunk 3]
```diff
@@ -197,28 +197,33 @@
   &.is--visible {
     opacity: 1;
     pointer-events: auto;
+    overflow-y: scroll;
```
Comment (hyu) [inline src/_sass/home.scss:200]: Realized that you couldn't see the bottom of the form on mobile! 😬

File: src/_includes/index_cta.html [hunk 4]
```diff
@@ -1,18 +1,88 @@
 <section class="modal" data-modal="one">
-  <h2 class="modal__title">
-    Schedule a Demo
-  </h2>
-  <button class="modal__close">{% svg assets/svg/x.svg %}</button>
-  <div class="modal__content">
-    <div class="calendly-inline-widget" data-url="https://calendly.com/dbentley/tilt-enterprise?hide_event_type_details=1">
+  <div class="calendly">
```
Comment (hyu) [inline src/_includes/index_cta.html:2]: One small nit: These classes are global, so we should add a prefix to `calendly` and `callback`...maybe the same `modal-` prefix?

**Comment by hyu** [comment]
Fixed layout: 

Screen Shot 2020-08-07 at 11 54 37 AM

File: src/_includes/index_cta.html (hunk 2 above)
Comment (jazzdan) [reply src/_includes/index_cta.html:50]: ohhh I didn't realize that was a thing, t\that's cool!

File: src/_includes/index_cta.html (hunk 4 above)
Comment (jazzdan) [reply src/_includes/index_cta.html:2]: Yeah totally, will do.

**Pull request merged**